src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
I see now, sorry I didn't think about it carefully enough, you're right. I actually think having `download` return an array would be an elegant way to handle it.